### PR TITLE
Fix modifiers

### PR DESCRIPTION
**Pull request by mdbenjam**
Currently modifiers are restricted by cleared EPs, and not restricted by pending EPs created outside the past thirty days. This PR selects the available modifiers by looking at all pending EPs. Also, adding links to the dev/set-users page to modify EP data.

**Test Plan**
- [ ] Verify manually by using EP data that contains a cleared EP, and an old pending EP and making sure they effect the available modifiers.
- [ ] Run test suite

### CODE REVIEW

File: app/models/appeal.rb [hunk 1]
```diff
@@ -216,22 +229,27 @@ def select_non_canceled_end_products_within_30_days(end_products)
...
+    # Find all pending EPs
+    end_products.select do |end_product|
+      end_product[:status_type_code] == "PEND"
+    end
+  end
+
+  def pending_eps
+    bgs = BGSService.new
+    end_products = Dispatch.filter_dispatch_end_products(
+      bgs.get_end_products(sanitized_vbms_id))
```
Comment (shanear) [inline app/models/appeal.rb]: should we memoize this? `bgs.get_end_products(sanitized_vbms_id)`

File: app/models/appeal.rb [hunk 2]
```diff
@@ -216,22 +216,29 @@ def select_non_canceled_end_products_within_30_days(end_products)
...
+  def select_pending_eps(end_products)
+    # Find all pending EPs
+    end_products.select do |end_product|
+      end_product[:status_type_code] == "PEND"
+    end
+  end
+
+  def bgs
+    @bgs ||= BGSService.new
+  end
```
Comment (mdbenjam) [inline app/models/appeal.rb]: Memoized bgs. I figure we can't really add a test for this method. Do you have any thoughts?

**Comment by mdbenjam** [comment]
Added the tests!

File: app/models/appeal.rb (hunk 2 above)
Comment (shanear) [inline app/models/appeal.rb]: we don't need it thats cool

File: spec/models/appeal_spec.rb [hunk 3]
```diff
@@ -429,6 +465,59 @@
...
+          status_type_code: "CAN"
+        },
+        {
+          claim_receive_date: last_year,
+          claim_type_code: "172BVAG",
+          status_type_code: "CLR"
+        }
+      ]
+    end
+    it do
```
Comment (shanear) [inline spec/models/appeal_spec.rb]: for multiline `it` statements we should have a string description.

**Pull request merged**